File: data/gcc_versions_comparison/src/program/src/func37.c
#include "program.h" 
array_t* func37(array_t_param* vars, int loopsFactor) {
   size_t pCounter = vars->size;
   unsigned int loop17 = 0;
   unsigned int loopLimit17 = (rand()%loopsFactor)/4 + 1;
   for(; loop17 < loopLimit17; loop17++) {
   }
   array_t* array50;
   if (pCounter > 0) {
      array50 = vars->data[--pCounter];
      array50->refC++;
      DEBUG_COPY(array50->id);
   } else {
      array50 = (array_t*)malloc(sizeof(array_t));
      array50->size = 859;
      array50->refC = 1;
      array50->id = 50;
      array50->data = (unsigned int*)malloc(array50->size*sizeof(unsigned int));
      memset(array50->data, 0, array50->size*sizeof(unsigned int));
      DEBUG_NEW(array50->id);
   }
   return array50;
}

File: data/multilang_comparison/program_julia/src/func37.jl
include("program_julia_head.jl")
function func37(vars::JArray_param, loopsFactor::Int)::JArray
   pCounter = length(vars.data)
   loopLimit17 = (50)/4 + 1
   for i in 1:loopLimit17 
   end
   array50 = nothing
   if pCounter > 1
       array50 = vars.data[pCounter]
       pCounter -= 1
       array50.refC += 1
   else
       array50 = JArray(zeros(Int, 859), 859, 1, 50)
   end
   return array50;
end

